tools/get_quad_ids.py

```diff
@@ -0,0 +1,52 @@
+from penquins import Kowalski
+import json
+
+
+def get_field_ids(catalog, field=301, ccd=2, quad=3, minobs=5, n=10):
+    '''Get ids for a specific quad of a CCD for a particular ZTF field.
```

Comment:
Can we have a real numpydocs docstring for this? I.e. of the style
https://github.com/skyportal/skyportal/blob/master/skyportal/handlers/api/observation_plan.py#L481

Should be quick given the info you have.